CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## Changelog
 
 ### 4.0.0 (Next)
+* [#176](https://github.com/alexa-js/alexa-app/pull/176): Added flexibility to attach alexa-app to express app or router - [@bgjehu](https://github.com/bgjehu).
```

Comment:
so before the change, users (whoever use the module) need to pass in both express app and express router, and endpoint to specify where to attach the router to the app. But there is no need to pass in both. If users don't use express router, but just an express app. they should be just attach to the app and go. And if they use express router, they wouldn't want to do attaching router to app through alexa-app module. They would desire just attach the alexa-apo to an express router. And they could handle the attaching router to app themselves outside alexa-app scope (meaning not thru alexa.express() function)